spec.html

```diff
@@ -6906,15 +6844,14 @@ <h1>
       <h1>
         CreateIterResultObject (
           _value_: unknown,
-          _done_: unknown,
+          _done_: a Boolean,
         )
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It creates an object that supports the IteratorResult interface.</dd>
+        <dd>It creates an object that conforms to the <i>IteratorResult</i> interface.</dd>
```

Comment:
Unrelated: we should dfn `IteratorResult interface`, and all the [other interface names](https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-common-iteration-interfaces), and stop italicizing them.

Reply:
I almost did that as part of this PR but backed it out. I can send a follow-up.